recipe/meta.yaml

```diff
@@ -38,13 +38,12 @@ requirements:
     - pthread-stubs
     - m2-patch  # [win]
   host:
-    - pkg-config
-    - gobject-introspection
+    - pkg-config 0.29.2
```

Comment:
Does it really need to be in host?

Reply:
yes, seems to be some meson hiccup. it can't find pkg-config to detect dependencies if not in host section. I don't know conda well enough yet to understand why.